README.md

```diff
@@ -69,6 +69,55 @@ To mark blocks of rules as targets for extraction use 'loud' (`!`) start and end
 
 Webpack will write 3 stylesheets:- `main.css`, `critical.css` and `comments.css`.
 
+## Advanced usage
+
+Blocks may be nested:
+
+```css
+/*! start:header.css */
+.header {
+  padding: 10px 20px;
+  background: #333;
+  color: #FFF;
+}
+/*! start:header-menu.css */
```

Comment:
I'll lookup the other issues. I implemented it on article ages ago too but
removed it again.

On Tuesday, 17 May 2016, Rhys Evans <notifications@github.com> wrote:

> In README.md
> <https://github.com/i-like-robots/extract-css-block-webpack-plugin/pull/1#discussion_r63505275>
> :
>
> > @@ -69,6 +69,55 @@ To mark blocks of rules as targets for extraction use 'loud' (`!`) start and end
> >
> >  Webpack will write 3 stylesheets:- `main.css`, `critical.css` and `comments.css`.
> >
> > +## Advanced usage
> > +
> > +Blocks may be nested:
> > +
> > +```css
> > +/*! start:header.css */
> > +.header {
> > +  padding: 10px 20px;
> > +  background: #333;
> > +  color: #FFF;
> > +}
> > +/*! start:header-menu.css */
>
> nice. I was just wondering how to approach main-ie8.css
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/i-like-robots/extract-css-block-webpack-plugin/pull/1/files/265ee21237764c34400411c5920c916158e333f7..615f8d8a8ac7d5a4ca2c643036474b8ab6544645#r63505275>
>